### DISCUSSION TITLE
Anchor link in Markdown does not work well on Mobile app for Android

### DISCUSSION THREAD

**Discussion by tez3998** [community/community#81528] [Mobile]
### Select Topic Area

Bug

### Body

# Problem I have encountered
In mobile app, clicking on an anchor link (often used in a table of contents) takes me to the top of the Markdown instead of to the desired location.
On the other hand, doing the same thing on Chrome works fine.

# My environments
- OS: Android
- Version of Android: 14
- Version of GitHub mobile app: 1.142.0

# Repositories where the problem can be reproduced
- [aws/aws-sdk-js-v3](https://github.com/aws/aws-sdk-js-v3)
- [donnemartin/system-design-primer](https://github.com/donnemartin/system-design-primer)

# Videos showing behavior
## Behavior on mobile app (not work well): 
You can see that clicking on any anchor links takes me to the top of Markdown.

https://github.com/community/community/assets/90051826/5d271f60-ff15-432c-b98b-55a21042bf8a

## Behavior on Google Chrome (works fine!):
Clicking on an anchor link takes me to the corresponding location.

https://github.com/community/community/assets/90051826/4bf14a07-d896-44da-9dc3-49cc92996b77

**Comment by SMGolamAlif** [comment]
Hey tez3998,

Looks like you've stumbled upon a glitch in the GitHub mobile app when it comes to anchor links, specifically in the table of contents. Let's see if we can troubleshoot this:

**Problem Description:**
In the mobile app (Android OS version 14), clicking on an anchor link redirects to the top of the Markdown instead of the intended location. However, the same action works fine on Google Chrome.

**Possible Solutions:**
1. **Update GitHub App:**
   Ensure that your GitHub mobile app is updated to the latest version (1.142.0). Sometimes, newer app versions address bugs and improve functionality.

2. **Browser Compatibility:**
   Since the issue doesn't occur in Chrome, it might be related to the mobile app's rendering. Consider reporting this behavior directly to GitHub support or through their feedback channels.

3. **Testing on Other Repositories:**
   Confirm if this problem is specific to certain repositories or if it occurs universally. You've mentioned examples, so maybe try a few more repositories to narrow down the scope.

4. **Alternative Markdown Features:**
   While waiting for a potential fix, you might explore alternative methods in your Markdown for navigation, such as using internal links or section headers.

**Reporting the Issue:**
Consider reporting this behavior to GitHub support or through the official channels. They might not be aware of this particular mobile app quirk, and your input could contribute to improving the GitHub experience for everyone.

If these suggestions don't resolve the problem, reaching out to the GitHub community or support would be the next steps.

Hope this helps in making those anchor links play nice on your mobile app! 📱🔗

**Reply by tez3998** [reply]
@SMGolamAlif 
Thank you for your suggestions.

This seems to be a problem with the mobile app`s rendering, so I don't think we can solve this problem on our own.
And, I believe this place is the official GitHub community.
